ipykernel/kernelspec.py

```diff
@@ -22,7 +22,7 @@
 RESOURCES = pjoin(os.path.dirname(__file__), 'resources')
 
 
-def make_ipkernel_cmd(mod='ipykernel', executable=None, extra_arguments=None, **kw):
+def make_ipkernel_cmd(mod='ipykernel_launcher', executable=None, extra_arguments=None, **kw):
```

Comment:
Why add a new module and change default behavior to use it, rather than just changing the behavior of `ipykernel.__main__`?